wsgi/bjoern.rst

```diff
@@ -0,0 +1,226 @@
+Bjoern
+======
+
+`bjoern <https://github.com/jonashaag/bjoern>`_ is a fast And ultra-lightweight HTTP/1.1 WSGI Server for CPython2 and 3 written in C.
+It claims to be the fastest, smallest and most lightweight WSGI server.
+Features:
+
+* ~ 1000 lines of C code
+* Memory footprint ~ 600KB
+* supports Python 2 and 3
+* Single-threaded
+* supports TCP `host:port` addresses and Unix sockets
+* Full persistent connection ("*keep-alive*") support in both HTTP/1.0 and 1.1,
+  including support for HTTP/1.1 chunked responses
```

Comment:
Remove marketing information of bjoern, and let the link to the repo suffice. This information can get out of date and adds a maintenance burden.